### ISSUE TITLE
Staging Review finding: Confirmation page focus should start on the success alert

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#104046]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Accredited Rep Facing (ARF) + Accredited Rep Mgmt (ARM)
**Product:** ARF (Accredited Rep Portal MVP) + ARM (Appoint a Rep 2.0)
**Feature:** Form 21-22: Digital Submission (ARM) and Accept/Decline (ARF)

## Findings details
**VA.gov Experience Standard - issue:** [User experiences predictable interactions with components and patterns.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Steps to reproduce
1. Go through Appoint 2.0 ending in digital submission
2. When the confirmation page loads, hit "Tab" on your keyboard. Observe that the focus is on the download link, not the green success alert. (The focus should be on the green focus alert)

### Description

When submitting to complete the flow for appointing an accredited representative, keyboard focus is lost as you transition to the confirmation page. This has two major consequences for accessibility:
- For sighted keyboard-only users, the viewport shows the top of the page and the success alert. However, if you press [tab] to interact with any elements on the page (eg. to download a PDF copy) you jump down to the footer of the page. This is disorienting, and it's a huge pain to shift-tab your way back up the page to where you wanted to be in the first place.
- For screen reader users who don't get the visual cue of the page change, nothing is announced by the screen reader with the page transition. Silence can be disorienting and confusing, and users may incorrectly guess that their submission didn't go through.

<details><summary>Link, screenshot or steps to recreate</summary>

![Confirmation page for "Request help from a VA accredited representative or VSO," with a green success alert reading "You’ve submitted your form to appoint the accredited representative on February 26, 2025."](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/99796-Brian-1.png?raw=true)

</details>

## Recommended action

Typically, I would expect focus to go to the success alert. That puts your next tab stop as something actionable, and by receiving focus the success alert will immediately be announced to screen reader users.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Keyboard
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by mtri** [comment]
Noticed a bug in Safari. When tabbing after the alert is auto-focussed on load, the focus only goes to the address in the widget and no other link. It seems to work fine in Chrome.

Steps to recreate:
1. Log in as vets.gov.user+124@gmail.com in Safari
2. Appoint Maryland Department of Veterans
3. When the confirmation page loads, hit tab
4. Observe that the focus immediately goes to the address link in the widget
